Pages/SearchPage.tsx

```diff
@@ -1,67 +1,298 @@
+        date: "Wednesday Nights 5-9pm",
+        image: "https://c8.alamy.com/comp/K70N7F/inside-the-libertine-brewing-companys-brew-pub-in-san-luis-obispo-K70N7F.jpg",
+        savedIcon: true,
+        filterCategories: ["Food", "Comedy", "Night Life", "Downtown"]
+    },
+    {
+        id: 3,
+        title: "Woodstock's Trivia Night",
+        date: "Monday Nights 9-11pm",
+        image: "https://slochamber.org/wp-content/uploads/2018/08/Woodstocks-Pizza-Backyard-3_1024.png",
+        savedIcon: true,
+        filterCategories: [
+            "Family Friendly",
+            "All Ages",
+            "Trivia",
+            "Night Life",
+            "Downtown"
+        ]
+    },
+    {
+        id: 4,
+        title: "Libertine Comedy Night",
+        date: "Wednesday Nights 5-9pm",
+        image: "https://c8.alamy.com/comp/K70N7F/inside-the-libertine-brewing-companys-brew-pub-in-san-luis-obispo-K70N7F.jpg",
+        savedIcon: true,
+        filterCategories: ["Food", "Comedy", "Night Life", "Downtown"]
+    },
+    {
+        id: 5,
+        title: "Woodstock's Trivia Night",
+        date: "Monday Nights 9-11pm",
+        image: "https://slochamber.org/wp-content/uploads/2018/08/Woodstocks-Pizza-Backyard-3_1024.png",
+        savedIcon: true,
+        filterCategories: [
+            "Family Friendly",
+            "All Ages",
+            "Trivia",
+            "Night Life",
+            "Downtown"
+        ]
+    },
+    {
+        id: 6,
+        title: "COYA Taco Night",
+        date: "Tuesday Nights 5-9pm",
+        image: "https://s3-media0.fl.yelpcdn.com/bphoto/kkR5Sb3WeGAAVRLC6dAIOQ/o.jpg",
+        savedIcon: false,
+        filterCategories: [
+            "Family Friendly",
+            "All Ages",
+            "Peruvian",
+            "Food",
+            "Deal"
+        ]
+    },
+    {
+        id: 6,
+        title: "COYA Taco Night",
+        date: "Tuesday Nights 5-9pm",
+        image: "https://s3-media0.fl.yelpcdn.com/bphoto/kkR5Sb3WeGAAVRLC6dAIOQ/o.jpg",
+        savedIcon: false,
+        filterCategories: [
+            "Family Friendly",
+            "All Ages",
+            "Peruvian",
+            "Food",
+            "Deal"
+        ]
+    },
+    {
+        id: 6,
+        title: "COYA Taco Night",
+        date: "Tuesday Nights 5-9pm",
+        image: "https://s3-media0.fl.yelpcdn.com/bphoto/kkR5Sb3WeGAAVRLC6dAIOQ/o.jpg",
+        savedIcon: false,
+        filterCategories: [
+            "Family Friendly",
+            "All Ages",
+            "Peruvian",
+            "Food",
+            "Deal"
+        ]
+    },
+    {
+        id: 6,
+        title: "COYA Taco Night",
+        date: "Tuesday Nights 5-9pm",
+        image: "https://s3-media0.fl.yelpcdn.com/bphoto/kkR5Sb3WeGAAVRLC6dAIOQ/o.jpg",
+        savedIcon: false,
+        filterCategories: [
+            "Family Friendly",
+            "All Ages",
+            "Peruvian",
+            "Food",
+            "Deal"
+        ]
+    }
+];
```

Comment:
Just an idea we can implement later but maybe put all data into a separate json file?